content/applications/hr/payroll.rst

```diff
@@ -130,8 +136,13 @@ by typing in the time.
    The :guilabel:`Work From` and :guilabel:`Work To` times must be in a 24-hour format. For example,
    `2:00 PM` would be entered as `14:00`.
 
-If the working time should be in a two-week configuration, click the :guilabel:`Switch To 2 Week
-Calendar` button. This creates entries for an :guilabel:`Even week` and an :guilabel:`Odd week`.
+If the working time should be in a two-week configuration, click the :guilabel:`Switch to 2 weeks
+calendar` button in the top left. This creates entries for an :guilabel:`Even week` and an
+:guilabel:`Odd week`.
+
+.. image:: payroll/new-working-schedule.png
+   :align: center
+   :alt: New working schedule form.
 
```

Comment:
I believe the employee has their company set and time zone set and just enters their time sheets as usual. So they know what working hours should be for them, as well as where they are located. I am going to put a pin in this since the other docs don't mention this in more detail. I asked Quentin about this. From what I believe, you'd enter your worked hours whatever they are for you (like 9-5) and Odoo should know where the employee is located on their card, and take the time for their work base don that. IF for some reason this is a complicated issue and needs a lot more explanation that what is here, I will come back and edit this, but since this was OK for the other versions and there was no issue form the product people on what I wrote, I will leave this alone for the moment.